### PR TITLE
postmap: add page

### PR DESCRIPTION
**Pull request by sakibjunaed**
Fixes issue #9475

- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Review by kbdharun — CHANGES_REQUESTED** [review]
Thanks for your contribution. This command must be placed in `common` directory as it works with both `linux` and macos `osx`.

File: pages/linux/postmap.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# postmap
+
+> Create or query one or more Postfix lookup tables, or updates an existing one.
```
Comment (kbdharun) [inline pages/linux/postmap.md:3]:
```suggestion
> Create or query one or more Postfix lookup tables, or update an existing one.
```

File: pages/linux/postmap.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+> More information: <https://manned.org/postmap>.
+
+- Search the specified maps for key and remove one entry per map:
```
Comment (kbdharun) [inline pages/linux/postmap.md:6]:
```suggestion
- Search specified maps for the key and remove one entry per map:
```
Is this better?

File: pages/linux/postmap.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+`postmap -d {{key}} {{input file}}`
```
Comment (kbdharun) [inline pages/linux/postmap.md:8]:
```suggestion
`postmap -d {{key}} {{input_file}}`
```

File: pages/linux/postmap.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+- Enable message header query mode:
+
+`postmap -h {{input file}}`
```
Comment (kbdharun) [inline pages/linux/postmap.md:12]:
```suggestion
`postmap -h {{input_file}}`
```

File: pages/linux/postmap.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+- Search the specified maps for key and write the first value found to the standard output stream:
```
Comment (kbdharun) [inline pages/linux/postmap.md:14]:
```suggestion
- Search specified maps for the key and write the first value found to the standard output stream:
```

File: pages/linux/postmap.md [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+
+`postmap -q {{key}} {{input file}}`
```
Comment (kbdharun) [inline pages/linux/postmap.md:16]:
```suggestion
`postmap -q {{key}} {{input_file}}`
```

File: pages/linux/postmap.md [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+
+- Retrieve all database elements, and write one line of key value output for each element:
+
+`postmap -s {{input file}}`
```
Comment (kbdharun) [inline pages/linux/postmap.md:20]:
```suggestion
`postmap -s {{input_file}}`
```

File: pages/common/postmap.md [hunk 8]
```diff
@@ -0,0 +1,24 @@
+# postmap
+
+> Create or query one or more Postfix lookup tables, or update an existing one.
+> More information: <https://manned.org/postmap>.
+
+- Search specified maps for the key and remove one entry per map:
```
Comment (EmilySeville7cfg) [inline pages/common/postmap.md:6]:
```suggestion
- Search specific maps for a key and remove one entry per map (output stream: `stdout`):
```

```suggestion
- Search specific maps for a key and remove one entry per map (output stream: `stderr`):
```

Optional fix: `specific` may sound better.
Optional fix: add the correct output stream name explicitly.

**Review by EmilySeville7cfg — APPROVED** [review]
At least one mandatory fix detected, changes requested.
Page should be scanned for multiple violations of similar flaws found (suggestions are created just for their first occurrences).
Thanks for a contribution! ❤️

**Review by EmilySeville7cfg — CHANGES_REQUESTED** [review]
- At least one mandatory fix detected, changes requested.
- Page should be scanned for multiple violations of similar flaws found (suggestions are created just for their first occurrences).
- Thanks for a contribution! heart

File: pages/common/postmap.md [hunk 9]
```diff
@@ -0,0 +1,24 @@
...
+
+`postmap -d {{key}} {{input_file}}`
```
Comment (pixelcmtd) [inline pages/common/postmap.md:8]: We're currently in the process of mandating that all paths start with `path/to/`.

```suggestion
`postmap -d {{key}} {{path/to/file}}`
```

File: pages/common/postmap.md [hunk 10]
```diff
@@ -0,0 +1,24 @@
...
+
+- Enable message header query mode:
+
+`postmap -h {{input_file}}`
```
Comment (pixelcmtd) [inline pages/common/postmap.md:12]:
```suggestion
`postmap -h {{path/to/file}}`
```

File: pages/common/postmap.md [hunk 11]
```diff
@@ -0,0 +1,24 @@
...
+
+- Search specified maps for the key and write the first value found to the standard output stream:
+
+`postmap -q {{key}} {{input_file}}`
```
Comment (pixelcmtd) [inline pages/common/postmap.md:16]:
```suggestion
`postmap -q {{key}} {{path/to/file}}`
```

File: pages/common/postmap.md [hunk 12]
```diff
@@ -0,0 +1,24 @@
...
+
+- Retrieve all database elements, and write one line of key value output for each element:
+
+`postmap -s {{input_file}}`
```
Comment (pixelcmtd) [inline pages/common/postmap.md:20]:
```suggestion
`postmap -s {{path/to/file}}`
```

**Pull request closed without merging**